src/Executables/Game/Extensions/ConfigurationExtensions.cs

```diff
@@ -6,6 +6,18 @@ public static class ConfigurationExtensions
 {
     public static void AddQuantumCoreDefaults(this IConfigurationBuilder config)
     {
+        // Empire Start Locations
+        config.AddInMemoryCollection(new Dictionary<string, string?>
+        {
+            {"empire:0:x", "475000"}, // Red
```

Comment:
I think if adding another empire entry at `index` 0 with x=,y=0 would make the code cleaner?
Wouldn't need to be subtracting 1 to the chosen empire this way.